lib/Monocle.php

```diff
@@ -32,11 +32,11 @@ public function get_active($eids, $minIv, $minLevel, $exMinIv, $bigKarp, $tinyRa
         }
         if (count($eids)) {
             $tmpSQL = '';
-            if(!empty($tinyRat) && $tinyRat === 'true' && ($key = array_search("19", $eids)) === false){
+            if (!empty($tinyRat) && $tinyRat === 'true' && ($key = array_search("19", $eids)) === false) {
```

Comment:
Would `in_array` be more appropriate here? I think it's marginally quicker if all you need to do is verify if something is in an array?

Reply:
It seems the difference is much more negligible in newer versions of PHP. Probably doesn't make much difference :)